src/main/java/uk/gov/companieshouse/service/dissolution/DissolutionEmailService.java

```diff
@@ -63,6 +65,17 @@ public void sendApplicationOutcomeEmail(Dissolution dissolution, DissolutionVerd
         sendEmail(emailDocument);
     }
 
+    public void sendPendingPaymentEmail(Dissolution dissolution) {
```

Comment:
Missing unit test for this

Reply:
@faggc If you need any help with unit tests this let me know